extensions/blocks/subscriptions/shared.scss

```diff
@@ -0,0 +1,69 @@
+.wp-block-jetpack-subscriptions {
+	position: relative;
+
+	.wp-block-jetpack-subscriptions__form,
+	form {
+		display: flex;
+		align-items: start;
+
+		.wp-block-jetpack-subscriptions__textfield .components-text-control__input,
+		.wp-block-jetpack-subscriptions__button,
+		input[type="email"],
+		button {
+			line-height: initial;
+		}
+
+		.wp-block-jetpack-subscriptions__button,
+		button {
+			border-style: solid;
+			border-color: transparent;
+		}
+
+		.wp-block-jetpack-subscriptions__textfield, p#subscribe-email {
+			flex-grow: 1;
+			background: transparent !important;
```

Comment:
Any way to avoid `!important`, e.g. by adding CSS specificity otherwise?

Reply:
Yes most likely, I will revisit this.